prisma/schema.prisma

```diff
@@ -80,32 +80,43 @@ model Job {
   isVerifiedJob      Boolean         @default(false) @map("is_verified_job")
   postedAt           DateTime        @default(now())
   updatedAt          DateTime        @updatedAt
-  user               User            @relation(fields: [userId], references: [id],onDelete: Cascade)
+  user               User            @relation(fields: [userId], references: [id], onDelete: Cascade)
 }
 
 model Experience {
-  id                Int @id @default(autoincrement())
-  companyName       String 
+  id                Int             @id @default(autoincrement())
+  companyName       String
   designation       String
   EmploymentType    EmployementType
   address           String
   workMode          WorkMode
   currentWorkStatus Boolean
   startDate         DateTime
   endDate           DateTime?
-  description       String  
-  userId            String 
-  user              User      @relation(fields: [userId] ,references: [id])
+  description       String
+  userId            String
+  user              User            @relation(fields: [userId], references: [id], onDelete: Cascade)
 }
 
 model Project {
-  id              Int @id @default(autoincrement())
-  projectName     String 
-  projectSummary  String
-  projectLiveLink String?
-  projectGithub   String
-  userId          String
-  user            User @relation(fields: [userId] , references: [id])
+  id               Int          @id @default(autoincrement())
+  projectName      String
+  projectThumbnail String?
+  projectSummary   String
+  projectLiveLink  String?
+  projectGithub    String
+  stack            ProjectStack @default(OTHERS)
+  userId           String
+  user             User         @relation(fields: [userId], references: [id], onDelete: Cascade)
+}
+
+enum ProjectStack {
+  GO
+  PYTHON
+  MERN_NEXTJS
```

Comment:
dont mix mern with nextjs, have them as seperate, dont add java springboot will be a good idea in replacement of java

Reply:
made changes for ProjectStack enums as well as wherever it's getting used